From 7d4528957f6e2930f8a86694d5a48df941228052 Mon Sep 17 00:00:00 2001
From: David Mouse <dmouse10462@gmail.com>
Date: Sun, 6 Sep 2020 01:33:52 -0400
Subject: [PATCH] Implement FreeDesktop tray icon for Flatpak

---
 jellyfin_mpv_shim/gui_mgr.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/jellyfin_mpv_shim/gui_mgr.py b/jellyfin_mpv_shim/gui_mgr.py
index 600582c..8bac7f7 100644
--- a/jellyfin_mpv_shim/gui_mgr.py
+++ b/jellyfin_mpv_shim/gui_mgr.py
@@ -405,6 +405,7 @@ def die():
 
         icon = Icon(USER_APP_NAME, menu=Menu(*menu_items))
         icon.icon = Image.open(get_resource("systray.png"))
+        icon.freedesktop_icon_name = os.environ.get("FLATPAK_ID") or None
         self.icon_stop = icon.stop
         def setup(icon):
             icon.visible = True
             